simulator/travis/minimesosFile

```diff
@@ -0,0 +1,291 @@
+minimesos {
+    clusterName = "Cook Simulation Cluster"
+    loggingLevel = "INFO"
+    mapAgentSandboxVolume = false
+    mapPortsToHost = false
+    mesosVersion = "1.0.0"
```

Comment:
unfortunately the newest one they support is 1.0.0.

Failed to run command 'up'. Failed to load cluster configuration from minimesosFile: Property 'mesosVersion' supports values: 0.25,0.25.0,0.26,0.27,0.28.0,0.28.1,0.28,1.0.0